playbooks/ufw-playbook.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- hosts: frontend_servers
+  roles:
+    - franklinkim.ufw
```

Comment:
As far as I understand the built-in ansible ufw changes rules in-memory but doesn't write them to files. Which would mean our rules would be lost after reboots.

If I'm misunderstanding we can definitely revisit though!